README.md

```diff
@@ -24,14 +24,14 @@ We welcome contributions! Many people all over the world have helped make this p
 
 * If you have a question or idea, [file a new issue](https://github.com/dotnet/runtime/issues/new).
 
-	* Is your feedback for .NET Framework? This repo is for .NET Core. If you are having issues with the "full" .NET Framework (also called "Desktop"), the best way to file a bug is the [Report a Problem](https://aka.ms/vs-rap) tool, which is integrated with the [VS Developer Community Portal](https://developercommunity.visualstudio.com/); or through [Product Support](https://support.microsoft.com/en-us/contactus?ws=support) if you have a contract.
+    * Is your feedback for .NET Framework? This repo is for .NET Core. If you are having issues with the "full" .NET Framework (also called "Desktop"), the best way to file a bug is the [Report a Problem](https://aka.ms/vs-rap) tool, which is integrated with the [VS Developer Community Portal](https://developercommunity.visualstudio.com/); or through [Product Support](https://support.microsoft.com/en-us/contactus?ws=support) if you have a contract.
 
 * If you want to contribute a fix or improvement, you'll want to check out these links
```

Comment:
I think we should delete this whole section. We have a CONTRIBUTING.MD file in the repo root for that (which I admit needs to cleaned up).

Reply:
Or just point to the contributing.md, like
```
## How can I contribute?

We welcome contributions! Many people all over the world have helped make this project better.

For further information see [Contributing](CONTRIBUTING.md).
```
?

Reply:
That sounds good